tests/cbmc/proofs/s2n_stuffer_printf/s2n_stuffer_printf_harness.c

```diff
@@ -21,6 +21,17 @@
 #include "stuffer/s2n_stuffer.h"
 #include "utils/s2n_mem.h"
 
+int nondet_int(void);
+
+int vsnprintf(char *str, size_t size, const char *fmt, va_list ap)
```

Comment:
How was vsnprintf implemented before this change? Where did the definition come from?

Reply:
There was no such implementation. CBMC's library doesn't ship it with 5.95.1 and will only start shipping an implementation with CBMC v6. So, really, the calls to `vsnprintf` were just ignored and a non-deterministic return value was produced.